praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/rest/NwkController.java

```diff
@@ -49,4 +49,15 @@ public List<NwkDto> getNwks(@RequestParam(name = "status", required = false) Str
         }
         return nwkService.findAllNwks();
     }
+
+    @PreAuthorize("hasRole('ROLE_' + T(de.muenchen.oss.praktikumsplaner.security.AuthoritiesEnum).AUSBILDUNGSLEITUNG.name())")
+    @PutMapping
+    public void updateNwk(@RequestBody NwkDto nwkDto) {
+        if (!nwkService.NwkExistsById(nwkDto.id())) {
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST,
```

Comment:
I think this should be a 404 Not Found, if we only allow modification and not creation via put we should tell the user that the resource he was trying to edit does not exist